### PR TITLE
Copy update on /download

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Copy update on `/download`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Please check page against [copy doc](https://docs.google.com/document/d/1egsCcmFiz8czeHsvwoB9Jz815Ij5Urtf7Ngrru6Zd4o/edit) and resolve comments
- View demo at : [https://ubuntu-com-10130.demos.haus/download](https://ubuntu-com-10130.demos.haus/download)

## Issue / Card

Fixes [#4298](https://github.com/canonical-web-and-design/web-squad/issues/4298)

## Screenshots
image

### CODE REVIEW

**Review by anthonydillon — APPROVED** [review]
LGTM 👍

**Comment by anthonydillon** [comment]
I can't seem to approve the suggestion though.

File: templates/download/index.html [hunk 1]
```diff
@@ -110,7 +110,7 @@ <h3>
         </a>
       </h3>
       <p>
-        Ubuntu is the reference OS for OpenStack. Try Canonical's OpenStack on a single machine or start building a production cloud on a cluster &mdash; just add servers.
+        Ubuntu is the reference OS for OpenStack. Try <a href='/openstack/install'>Canonical OpenStack</a> on a single machine or start building a production cloud on a cluster &mdash; just add servers.
```
Comment (carkod) [inline templates/download/index.html:113]: I think in HTML the convention is to use double quotes instead of single quotes.

Comment (minkyngkm) [reply templates/download/index.html]: Thanks Carlos for reviewing! Fixed!

**Pull request merged**